Revised the json response handling

amcguiga opened the pull request.

Schmidt-A commented:
Dec 26 needful banter results:
- move role checking/authorization to BaseView (is_admin, is_owner, etc)
- move json response generation to respective model objects
  - As part of this, define interface methods in meta.py/Base model (just have their body be a "raise NotImplementedError)
- update character inventory routes (either make both /items/{id} + /items or /inventory)

amcguiga commented:
I changed everything to items rather than leaving any vestiges of Inventory. If you disagree, I could change it back.

I implemented the payload methods in character, items, and accounts.

amcguiga commented:
This is ready to merge if y like it.
